services/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 if(SYSTEMD_FOUND)
     configure_file("${CMAKE_CURRENT_SOURCE_DIR}/sddm.service.in" "${CMAKE_CURRENT_BINARY_DIR}/sddm.service")
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sddm.service" DESTINATION "${SYSTEMD_SYSTEM_UNIT_DIR}")
+    
+    if(SMOOTH_PLYMOUTH_TRANSITION)
```

Comment:
Better, but it should probably install either one or the other service, the destination file name should also be the same.

Reply:
Okay. I'll change that right now.

Reply:
Fixed.